src/libs/modulegroups.c

```diff
@@ -975,32 +976,67 @@ static GList *_preset_from_string(gchar *txt)
 
 void init_presets(dt_lib_module_t *self)
 {
+  // all modules
   gchar *tx = NULL;
-  tx = dt_util_dstrcat(
-      tx, "ꬹ1ꬹ%s|%s||%s", _("technical"), "technical",
-      "colorin|hazeremoval|filmicrgb|clipping|flip|lens|exposure|denoiseprofile|demosaic|highlights");
-  tx = dt_util_dstrcat(tx, "ꬹ%s|%s||%s", _("grading"), "grading", "rgblevels|colorbalance|toneequal|temperature");
-  tx = dt_util_dstrcat(tx, "ꬹ%s|%s||%s", _("effects"), "effect", "sharpen|bilat");
-  dt_lib_presets_add(_("default"), self->plugin_name, self->version(), tx, strlen(tx), TRUE);
```

Comment:
The "default" preset is use in many place in the code as a fallback in case no valid preset is selected (first start, deletion of current preset, etc...) so we have to decide which is the new "default" preset and change the code in that places too.

Reply:
Maybe we should default to "scene referred" preset ? Maybe using a #define to specify the default preset. Do you have a list of all places in code where we reference it ?